packages/eventual-send/src/index.js

```diff
@@ -51,7 +51,7 @@ export function makeHandledPromise(Promise) {
   let presenceToHandler;
   let presenceToPromise;
   let promiseToUnsettledHandler;
-  let promiseToPresence; // only for HandledPromise.unwrap
+  let promiseToPresence;
```

Comment:
That old comment was correct when written. But trying to remove `promiseToPresence` reveals that it has since grown another use --- in shortening.